### DISCUSSION TITLE
commande HY type EP

### DISCUSSION THREAD

**Discussion by jeremGi** [cmcgerty/Sofar2mqtt#75] [Q&A]
bonjour

j'utilise depuis des mois cette version de sofar2mqtt qui fonctionne super bien en lecture des infos.
Depuis quelques jours je tente de piloter l'onduleur depuis jeedom via ce module.
pas de soucis pour les commandes standby , charge et décharge tout fonctionne mais il m'est impossible de passer en mode auto ou batterie save.
j'ai glané des infos a droite et a gauche et j'ai trouve que c'est le registre 0x1110 qui change en fonction du mode de batterie choisi.

si une âme charitable pouvait me venir en aide se serait super.

merci d'avance
j'arrive a le lire mais pas a l'ecrire et je dois avouer que la ca dépasse de beaucoup mes compétences.

**Comment by IgorYbema** [comment]
This is already possible in my sofar2mqtt v3.5 version:
mqtt: sofar2mqtt/set/mode_control 
"Energy storage working mode setting;
0: Spontaneous self-use mode
1: Time-of-use electricity price model
2: Timing charging and discharging mode
3: Passive mode
4: Peak clipping mode
5: Off-grid mode
Used to change the working mode."

**Reply by Fabkiki** [reply]
Bonjour,
j'utilise la version 3.6 mais j'ai le meme problème que @jeremGi  tout fonctionne sauf les commandes Auto et battery_save, aucune réaction de l'onduleur (HYD6000EP) les commandes sons ignoré merci si vous pouvez nous venir en aide.

**Comment by jeremGi** [comment]
Merci je 'avais pas la dernière version 😅
